flask/app/tissue_samples.py

```diff
@@ -143,7 +172,11 @@ def update_tissue_sample(id: int):
     if user_id:
         tissue_sample = (
             models.TissueSample.query.filter_by(tissue_sample_id=id)
-            .options(contains_eager(models.TissueSample.datasets))
+            .options(
+                contains_eager(models.TissueSample.datasets),
+                # contains_eager(models.TissueSample.created_by),
+                # contains_eager(models.TissueSample.updated_by),
```

Comment:
I had a careful look and I think it's okay for this PR if we don't return the users for the nested objects. If needed we can do it in a more focused PR.

Reply:
Right now the nested objects do return users, I thought the issue was that n+ 1 select statements are being generated as a result of inefficient joining?

eg. tissue samples

```
curl localhost:5000/api/tissue_samples/1\?user=1
{
  "created": "Mon, 07 Dec 2020 15:43:25 GMT", 
  "created_by": "admin", 
  "datasets": [
    {
      "batch_id": null, 
      "capture_kit": null, 
      "condition": "GermLine", 
      "created": "Mon, 03 Feb 2020 00:00:00 GMT", 
      "created_by": "admin", 
      "dataset_id": 1, 
      "dataset_type": "RGS", 
      "extraction_protocol": "Something", 
      "input_hpf_path": null, 
      "library_prep_date": null, 
      "library_prep_method": null, 
      "notes": "hello", 
      "read_length": null, 
      "read_type": null, 
      "sequencing_centre": "CHEO", 
      "sequencing_date": null, 
      "sequencing_id": null, 
      "tissue_sample_id": 1, 
      "updated": "Wed, 09 Dec 2020 19:45:55 GMT", 
      "updated_by": "admin"
    }
  ], 
  "extraction_date": "Sat, 04 Jan 2020 00:00:00 GMT", 
  "notes": "hello", 
  "participant_id": 1, 
  "tissue_processing": null, 
  "tissue_sample_id": 1, 
  "tissue_sample_type": "Blood", 
  "updated": "Fri, 11 Dec 2020 20:29:31 GMT", 
  "updated_by": "admin"
}
```